tests/test_citations.py

```diff
@@ -249,16 +249,56 @@ def test_cite_command_file(tmpdir):
 2. Vagelos Report Summer 2017
 Michael Zietz
 Figshare (2017) https://doi.org/gdz6dd
+DOI: 10.6084/m9.figshare.5346577.v1
+
+3. Opportunities and obstacles for deep learning in biology and medicine.
+Travers Ching, Daniel S Himmelstein, Brett K Beaulieu-Jones, Alexandr A Kalinin, Brian T Do, Gregory P Way, Enrico Ferrero, Paul-Michael Agapow, Michael Zietz, Michael M Hoffman, … Casey S Greene
+Journal of the Royal Society, Interface (2018-04) https://www.ncbi.nlm.nih.gov/pubmed/29618526
+DOI: 10.1098/rsif.2017.0387 · PMID: 29618526 · PMCID: PMC5938574
 '''
 
 references_markdown = '''\
 1. **Generalization of the Fermi Pseudopotential**  
 Trang T. Le, Zach Osman, D. K. Watson, Martin Dunn, B. A. McKinney  
+  
```

Comment:
I think our updated CSL style is causing these extra line breaks in markdown output. Will have to investigate. For now I just updated the test with this undesirable ouput.